verifyFiles.py

```diff
@@ -39,21 +39,26 @@ def get_dir_dict(directory):
 # Returns a dictionary where the keys are the directories in
 # 'condition_dir' and the values are the directory dictionaries (as
 # returned by get_dir_dict) associated to these directories.
-def get_condition_dict(condition_dir):
+def get_condition_dict(condition_dir,exclude_folders):
     condition_dict={}
-    for subject_name in os.listdir(condition_dir):
+    subject_names=[]
+    if exclude_folders is not None:
+            subject_names[:]=[subject_name for subject_name in os.listdir(condition_dir) if subject_name not in exclude_folders]
+    for subject_name in subject_names:
         subject_dir_path=os.path.join(condition_dir,subject_name)
         if os.path.isdir(subject_dir_path):
```

Comment:
This happens at the initial stage where the dictionary is getting created by going through every folder in every subject under every condition.Should we remove this ?